analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/FluentTestHelper.cs

```diff
@@ -41,7 +41,7 @@ internal static class FluentTestHelper
              where TAssertions : SelfReferencingCollectionAssertions<T, TAssertions>
         {
             _ = expected ?? throw new ArgumentNullException(nameof(expected));
-            self.Subject.Should().ContainInOrder(expected).And.HaveSameCount(expected);
+            self.Subject.Should().ContainInOrder(expected).And.BeEquivalentTo(expected).And.HaveSameCount(expected);    // BeEquivalentTo to have better collection message
```

Comment:
is `HaveSameCount` needed anymore? won't `BeEquivalentTo` fail if the numbers differ?